edx_proctoring/api.py

```diff
@@ -205,6 +205,19 @@ def get_review_policy_by_exam_id(exam_id):
     return ProctoredExamReviewPolicySerializer(exam_review_policy).data
 
 
+def _get_review_policy_by_exam_id(exam_id):
+    """
+    Looks up exam by the Primary Key. Returns None if not found
```

Comment:
So-small-its-barely-worth-mentioning-nit: `Primary Key` is a weird thing to capitalize in this docstring